routes/experimenters.js

```diff
@@ -30,3 +31,39 @@ exports.create = function (req, res) {
       });
    });
 };
+
+function updateExperimenter(req, res, exp) {
+    Experimenter.update({ _id : ObjectId(req.params.id) }, exp, function (err, numUpdated, rawRes) {
+       if (err) {
+          log.error('Error on PUT /experimenters/' + req.params.id, err);
+          return res.send(500);
+       }
+
+       if(numUpdated == 0) {
+           // no experimenters updated, bad PUT request
+           return res.send(400);
+       }
+
+       return res.status(200).send(rawRes);
```

Comment:
For successful PUT requests I prefer to send a 204 without content, like so:
    return res.send(204);